app/views/directives/deploy-image.html

```diff
@@ -60,6 +60,10 @@
                 </button>
               </span>
             </div>
+            <div ng-if="input.selectedProject.metadata.uid && !import.namespace" class="help-block">To pull a secured container image that is not from the internal registry, you must
+              <a href=""
+                ng-click="openCreateWebhookSecretModal()">create an image pull secret</a>
+               from your Docker credentials and add it to your service account. <a href="https://docs.openshift.org/latest/dev_guide/managing_images.html#using-image-pull-secrets" target="_blank"><span class="learn-more-inline">Learn More&nbsp;<i class="fa fa-external-link" aria-hidden="true"></i></span></a></div>
```

Comment:
Hmm, something related to wording. Seems it is better to use `your registry credentials` or `your image registry credentials` or whatever that avoids string `Docker` (similar to bug 1447203)?

Reply:
Thanks yeah, let's make this

"To deploy an image from a private repository, you must create an image secret with your image registry credentials."